Team417_PowerPlay/src/main/java/org/firstinspires/ftc/team417_PowerPlay/MiddleJunctionParkRightSide.java

```diff
@@ -0,0 +1,73 @@
+package org.firstinspires.ftc.team417_PowerPlay;
+
+import com.acmerobotics.roadrunner.geometry.Pose2d;
+import com.acmerobotics.roadrunner.trajectory.Trajectory;
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+
+@Autonomous(name="Interleague Right")
+public class MiddleJunctionParkRightSide extends BaseAutonomous {
```

Comment:
I feel like you have a lot of repeated code in all these Autonomous functions. You should consider doing what you can to combine them. Maybe take sections and move them into a helper function in `BaseAutonomous`.

Reply:
Will make another pull request